README.md

```diff
@@ -5,90 +5,71 @@
+You are required to _complete everything_ in this prep course: Without doing it, you may fall behind in class.
 
 We ask that you set aside a total of 40 hours for this prep course.
 
-**Touchpoint meetings:**
+**Touchpoint meetings:**  
 We'll be contacting you regularly to setup Skype/Google Hangout conversations to see how things are going.
 
-**Due Date:**
+**Due Date:**  
 The prep course is due 2 weeks before the bootcamp starts. This allows for time to give you feedback on the submitted work to make sure you are ready for Day 1.
 
-**Submitting Your Work:**
+**Submitting Your Work:**  
 As you are progressing through the course you will be required to submit various assignments. Here is the list of assignments that will require submission.
 
-  - FizzBuzz
-  - Two doors
-  - Toyota's are Cars too!
-  - Find the Largest Number in an Array
-
-For each of these, please submit the Github URL to us *(Git is explained in section 4)*.
+* FizzBuzz
+* Two doors
+* Toyota's are Cars too!
+* Find the Largest Number in an Array
 
-Vancouver students please email <ios-prep-vancouver@lighthouselabs.ca> with the URLs.
+For each of these, please submit the Github URL to us _\(Git is explained in section 4\)_.
 
-Toronto students please email <ios-prep-toronto@lighthouselabs.ca> with the URLs.
+Vancouver students please email [ios-prep-vancouver@lighthouselabs.ca](mailto:ios-prep-vancouver@lighthouselabs.ca) with the URLs.
 
-**Important:** Please include your Name and Cohort in the subject of the email (ie: _Prep Submission -- John Doe, Jan 11, 2016_).
+Toronto students please email [ios-prep-toronto@lighthouselabs.ca](mailto:ios-prep-toronto@lighthouselabs.ca) with the URLs.
 
+**Important:** Please include your Name and Cohort in the subject of the email \(ie: _Prep Submission -- John Doe, Jan 11, 2016_\).
 
 ##### Where to Get Help
-If you don't understand or get stuck on something, as a registered Lighthouse Labs student you have (or will soon receive) access to our private [iOS Prep Discussion Forum](http://ios-prep.lighthouselabs.ca/login). Please contact the admissions person you are in contact with if you feel you should have access but have not received it yet.
 
-This private discussion forum exists solely for students who are preparing for the bootcamp program to get help from instructors, mentors and each other on the prep course. You can submit your question as a new Topic (one problem per topic please; feel free to create multiple topics if you are stuck in multiple places). You will get an e-mail notification for every reply to your topic.
+As a registered Lighthouse Labs student, if you don't understand or get stuck on something,  have access to our private [iOS Prep Discussion Forum](http://ios-prep.lighthouselabs.ca/login). Please contact the admissions person you've been in contact with to gain forum access if you have not yet received.
```

Comment:
grammarm, modify to:
As a registered Lighthouse Labs student, if you don't understand or get stuck on something, you have access to our private [iOS Prep Discussion Forum](http://ios-prep.lighthouselabs.ca/login). Please contact the admissions coordinator you've been in contact with to get forum access.